README.md

```diff
@@ -39,6 +39,7 @@ shows which versions of each generator are inside the docker image.
 
 | Docker image           | uniffi-rs version     | uniffi-bindgen-cs version | uniffi-bindgen-go version | uniffi-bindgen-cpp version |
 |------------------------|-----------------------|---------------------------|---------------------------|----------------------------|
+| v0.25.0-10             | v0.3.0+0.25.0         | v0.8.1+v0.25.0            | v0.2.1+v0.25.0            | **v0.6.1+v0.25.0**         |
```

Comment:
Please bolden `uniffi-bindgen-cs`, and unbolden `uniffi-bindgen-cpp`. It makes it easier to see what generator versions were changed.

Reply:
Nice catch. Should be fixed now